package.json

```diff
@@ -37,9 +39,9 @@
     "autoprefixer": "10.4.5"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "CI=false react-scripts build",
-    "test": "react-scripts test",
+    "start": "craco start",
+    "build": "CI=false craco build",
+    "test": "craco test",
```

Comment:
could we please do it in a separate PR?

Reply:
Do you mean committing the same code but without updating `package.json`?
I'm not sure the code will work without this change...

Reply:
I mean create another branch, and make the `package.json` changes there.
why those changes are needed?

Reply:
why not ``import(`./busIcons/${id}.svg`)``?
could be more simple

Reply:
I already tried that, but it didn't work properly. I could give it another try